docs/src/calling-contracts/variable-outputs.md

```diff
@@ -1,17 +1,29 @@
-# Variable outputs
+# Output variables
```

Comment:
Shouldn't it be based on `VariableOutput`, i.e., keep it as `Variable outputs` (considering the client to be the source of truth here https://github.com/FuelLabs/fuel-core/blob/a3f584f12bb3e160aa00e08616a27c81734ebd02/crates/fuel-core/src/schema/tx/output.rs#L84)?

Reply:
Yup, I got confused